### PR TITLE
Set Livestream claim release time to start time from API

### PR DESCRIPTION
**Pull request by ktprograms**
## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [x] I have checked that this PR does not introduce a breaking change

## PR Type

What kind of change does this PR introduce?

- [x] Feature

## Fixes

Issue Number: #232 (`For the livestream content date - use the started at from livestream api (and not claim date)`)

## What is the current behavior?

Claim publish date used.

## What is the new behavior?

Date from `Start` field from Livestream API used.

### CODE REVIEW

File: app/src/main/java/com/odysee/app/model/Claim.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
 import java.text.NumberFormat;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
+import java.time.ZonedDateTime;
```
Comment (kekkyojin) [inline app/src/main/java/com/odysee/app/model/Claim.java:21]: ZonedDateTime was added in Android API Level 26 and Odysee Android must run on devices running API Level 21 or higher

**Review by kekkyojin — CHANGES_REQUESTED** [review]
You will need to use a different approach

File: app/src/main/java/com/odysee/app/model/Claim.java (hunk 1 above)
Comment (ktprograms) [reply app/src/main/java/com/odysee/app/model/Claim.java:21]: Does this fix it: https://stackoverflow.com/a/60687913/15603854 (this bit of code is currently in the project)

I didn't get compile errors like I normally do with other API Level problems, so I think it's related to this.

Comment (kekkyojin) [reply app/src/main/java/com/odysee/app/model/Claim.java:21]: Oh. Sorry. You are right.

File: app/src/main/java/com/odysee/app/model/Claim.java [hunk 2]
```diff
@@ -444,13 +445,17 @@ public static Claim fromJSONObject(JSONObject claimObject) {
...
      * @return A new Claim object
      */
-    public static Claim fromLiveStatus(String claimId, String liveUrl, int livestreamViewers) {
+    public static Claim fromLiveStatus(String claimId, String liveUrl, int livestreamViewers, String startTime) {
         Claim claim = new Claim();
         claim.setClaimId(claimId);
         claim.setLivestreamUrl(liveUrl);
         claim.setLivestreamViewers(livestreamViewers);
+        StreamMetadata value = new StreamMetadata();
+        value.setReleaseTime(ZonedDateTime.parse(startTime).toInstant().getEpochSecond());
```
Comment (kekkyojin) [inline app/src/main/java/com/odysee/app/model/Claim.java:457]: Catch DateTimeParseException here and everywhere you used parse(). printStackTrace the exception. Return null

**Review by kekkyojin — DISMISSED** [review]
Catching exceptions have helped in prior debugging of bugs. Not catching them made some bugs hard to debug.

**Comment by ktprograms** [comment]
Diff [b00a756...4e1f64a](https://github.com/OdyseeTeam/odysee-android/compare/b00a756484020b69fd5b4ea8cc1ce918a6dede62..4e1f64a9d099b450de31acf96b0dcf040893b54c)
Catch exception and handle null

**Review by kekkyojin — APPROVED** [review]
This PR does what it means to do.

**Pull request merged**